azext_iot/sdk/iothub/service/operations/modules_operations.py

```diff
@@ -550,13 +555,12 @@ def invoke_method(
             header_parameters['accept-language'] = self._serialize.header("self.config.accept_language", self.config.accept_language, 'str')
 
         # Construct body
-        # @digimaun - Originally 'CloudToDeviceMethod'. Model serialization forces a null payload property to be removed.
-        # TODO: Test model behavior in latest autorest generator.
-        body_content = self._serialize.body(direct_method_request, 'object')
```

Comment:
We should keep the customizations unless proven otherwise. The comment explains the past reason for keeping as 'object'.